cmd/cli/plugin/cluster/osimage_oracle.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/aunum/log"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	k8sjson "k8s.io/apimachinery/pkg/runtime/serializer/json"
+	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	"github.com/vmware-tanzu/tanzu-framework/apis/run/v1alpha3"
+	"github.com/vmware-tanzu/tanzu-framework/cli/core/pkg/carvelhelpers"
```

Comment:
Plugins should not use `cli/core` module as a dependency. Please use `tkg/carvelhelpers` package within `tkg` module. You might need to do changes done under `cli/core/pkg/carvelhelpers/registry.go` to `tkg/carvelhelpers/registry.go`

Reply:
fixed